CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
 
+## [6.0.0] - 2021-12-13
```

Comment:
This should be a link (see all headings prior to 5.0.0 - 5.0.0 should also be a link).

Reply:
Good spot. Resolved.